src/api/entities/Identity/IdentityAuthorizations.ts

```diff
@@ -51,4 +53,41 @@ export class IdentityAuthorizations extends Authorizations<Identity> {
       next,
     };
   }
+
+  /**
+   * Retrieve a single Authorization Request targeting or issued by this Identity by its ID
+   *
+   * `authorizations` storage only returns results for the authorization target, so `authorizationsGiven` needs to be queried first to find the relevant target if present
```

Comment:
This seems more of an implementation detail that should be in a comment inside the method rather than in the JSDoc